examples/bert_ptq_cpu/mlflow_bert_config.json

```diff
@@ -0,0 +1,139 @@
+{
+    "azureml_client": {
```

Comment:
The bert example directory structure has changed. Please adapt new structure.
Also the mflow_bert_config.json should only contain configurations that are actually used by the mlflow example. For example if AML is not used then don't include azureml_client config. And so on for other config options also.
All example config should be as minimal as possible.